Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -164,6 +164,63 @@ function vtkMouseRangeManipulator(publicAPI, model) {
     const size = glRenderWindow.getViewportSize(renderer);
     // rescale size to match mouse event position
     model.containerSize = size.map((v) => v * ratio);
+
+    if (model.usePointerLock && !interactor.isPointerLocked()) {
+      interactor.requestPointerLock();
+      publicAPI.startPointerLockInteraction(interactor);
+    }
+  };
+
+  publicAPI.onButtonUp = () => {
+    publicAPI.exitPointerLock();
+  };
+
+  publicAPI.bindEvents = (container) => {
+    document.addEventListener(
+      'pointerlockchange',
+      publicAPI.handlePointerLockChange
+    );
+  };
+
+  //--------------------------------------------------------------------------
+
+  publicAPI.startPointerLockInteraction = (interactor) => {
+    // TODO: at some point, this should perhaps be done in
+    // RenderWindowInteractor instead of here.
+    // We need to hook into mousemove directly for two reasons:
+    // 1. We need to keep receiving mouse move events after the mouse button
+    //    is released. This is currently not possible with
+    //    vtkInteractorStyleManipulator.
+    // 2. Since the mouse is stationary in pointer lock mode, we need the
+    //    event.movementX and event.movementY info, which are not currently
+    //    passed via interactor.onMouseMove.
+    document.addEventListener('mousemove', publicAPI.onPointerLockMove);
+
+    let subscription = null;
+    const endInteraction = () => {
+      document.removeEventListener('mousemove', publicAPI.onPointerLockMove);
+      subscription?.unsubscribe();
+    };
+    subscription = interactor?.onEndPointerLock(endInteraction);
+  };
+
+  publicAPI.onPointerLockMove = (e) => {
+    model.previousPosition.x += e.movementX;
+    model.previousPosition.y += e.movementY;
+
+    publicAPI.onMouseMove(null, null, model.previousPosition);
```

Comment:
In thinking about this, I'm thinking that we can instead cache the interactor and renderer in onButtonDown, and used those to pass into onMouseMove here.

Reply:
And then clear said cache in onButtonRelease/Up (I forget what it's called).

Reply:
Why do we care if they are not used by onMouseMove?

Reply:
In thinking generally, the caching pattern would be useful. But you are right, they aren't used in MouseRangeManipulator.onMouseMove, so keeping the nulls is fine.